### ISSUE TITLE
Cbsodata 1.3.2 crashing in debug mode. 

### ISSUE THREAD

**Issue by TheHunter896** [J535D165/cbsodata#16]
**Problem**

When importing cbsodata (version 1.3.2) in debug mode script execution crashes.

**Where it happens**

Pycharm: 

- Python 3.7.3 (debug mode) 
- Python 3.7.8 (debug mode) 
- Haven't tested any other versions

Spyder:

- Python 3.8 (normal execution and debug mode)

**Expected behaviour**

Package is imported successfully.

**Pycharm error message**
Pycharm Community Edition (latest) + Python 3.7.3: Stack overflow error. (Same error in Python 3.7.8)

```python 

Fatal Python error: Cannot recover from stack overflow.


Thread 0x00004d14 (most recent call first):
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 300 in wait
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 552 in wait
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\pydevd.py", line 142 in _on_run
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\_pydevd_bundle\pydevd_comm.py", line 213 in run
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917 in _bootstrap_inner
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 885 in _bootstrap

Thread 0x000068f8 (most recent call first):
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\_pydevd_bundle\pydevd_comm.py", line 283 in _on_run
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\_pydevd_bundle\pydevd_comm.py", line 213 in run
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917 in _bootstrap_inner
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 885 in _bootstrap

Thread 0x00007ae0 (most recent call first):
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 300 in wait
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\queue.py", line 179 in get
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\_pydevd_bundle\pydevd_comm.py", line 358 in _on_run
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2019.2.2\helpers\pydev\_pydevd_bundle\pydevd_comm.py", line 213 in run
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 917 in _bootstrap_inner
  File "C:\Users\foobar\AppData\Local\Programs\Python\Python37\lib\threading.py", line 885 in _bootstrap

Current thread 0x00002a50 (most recent call first):
  File "C:\Users\foobar\PycharmProjects\powerbi_to_pdf\venv\lib\site-packages\cbsodata\cbsodata3.py", line 84 in __getattr__
  [Continues multiple times]
```

**Spyder error message**
```python
Kernel died...
```

**To reproduce the error**

Install cbsodata via pip ``` pip install cbsoadata ``` 

main.py 
```python
import cbsodata as cbs
```
Run main.py in debug mode. 
 
Run the same line in the versions and editors specified on top, same error.

**Pip freeze** 
```shell
cbsodata==1.3.2
certifi==2020.11.8
chardet==3.0.4
idna==2.10
numpy==1.19.0
pandas==1.1.4
pyodbc==4.0.30
python-dateutil==2.8.1
pytz==2020.4
PyYAML==5.3.1
requests==2.24.0
selenium==3.141.0
six==1.15.0
urllib3==1.25.11
``` 

Could it be my fault somehow?

**Comment by J535D165** [comment]
Thanks for reporting. Another user reported something similar. The main dependency of cbsodata is 'requests'. Can you import this dependency?

It might be something with Gevent (https://www.jetbrains.com/help/pycharm/debugger-python.html)

**Comment by wbijster** [comment]
I have a similar issue when importing cbsodata within Jupyter Notebooks: the kernel stops and is restarted.

```
import requests
import cbsodata

python 3.8.5
jupyter_client 6.1.7
cbsodata 1.3.3
```

It works fine with this configuration:

```
python 3.7.9
jupyter_client  6.1.7 
cbsodata 1.3.3
```

**Comment by J535D165** [comment]
I think this is related to #17. Is numpy or pandas imported somewhere? Can you confirm?

I think this issue is not on our side. The API was stable for a long time.

**Comment by wbijster** [comment]
Thanks, yes, numpy and pandas are imported, sorry, I did not notice that issue. 
FYI 
```
Python 3.8.5 environment:
numpy                     1.19.2
pandas                    1.1.3

Python 3.7.9 environment:
numpy                     1.19.5
pandas                    1.2.0
```

(I also did not expect that numpy and pandas would be more recent in my environment with the older python version, this was done by a generic anaconda install statement without specifying the package versions).

**Comment by galamit86** [comment]
Same issue with cbsodata V1.3.3 using VScode debug mode. Monkey-patching the removal of `__get_attr__`, as stated in #19 - does the trick as far as I can see.

**VS Code**

Version: 1.53.0
Commit: 8490d3dde47c57ba65ec40dd192d014fd2113496
Date: 2021-02-03T16:01:04.046Z (1 wk ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 19.6.0

**Python** version: 3.8.5


Terminal output:
```
Traceback (most recent call last):
  File "_pydevd_bundle/pydevd_cython.pyx", line 1557, in _pydevd_bundle.pydevd_cython.ThreadTracer.__call__
RecursionError: maximum recursion depth exceeded while calling a Python object
Fatal Python error: Cannot recover from stack overflow.
Python runtime state: initialized

Thread 0x000070000de9b000 (most recent call first):
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 306 in wait
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 558 in wait
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 232 in _on_run
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py", line 46 in run
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 890 in _bootstrap

Thread 0x000070000ce98000 (most recent call first):
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 306 in wait
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 558 in wait
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/pydevd.py", line 186 in _on_run
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py", line 46 in run
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 890 in _bootstrap

Thread 0x000070000be95000 (most recent call first):
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 302 in wait
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 558 in wait
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_timeout.py", line 43 in _on_run
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py", line 46 in run
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 890 in _bootstrap

Thread 0x000070000ae92000 (most recent call first):
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py", line 210 in _read_line
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py", line 228 in _on_run
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py", line 46 in run
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 890 in _bootstrap

Thread 0x0000700009e8f000 (most recent call first):
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 306 in wait
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/queue.py", line 179 in get
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py", line 339 in _on_run
  File "/Users/tslilstrauss/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_daemon_thread.py", line 46 in run
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/Users/tslilstrauss/.pyenv/versions/3.8.5/lib/python3.8/threading.py", line 890 in _bootstrap

Current thread 0x0000000108147dc0 (most recent call first):
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 84 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  File "/Users/tslilstrauss/Projects/statline-bq/.venv/lib/python3.8/site-packages/cbsodata/cbsodata3.py", line 85 in __getattr__
  ...
```

**Comment by J535D165** [comment]
Thanks to @nieuwenhoven, this issue was resolved in #19. See version 1.3.4 on PyPI for the fixed release.
